src/hooks/useKeyToAsset.ts

```diff
@@ -1,8 +1,11 @@
+import { IdlAccounts } from '@coral-xyz/anchor'
 import { useAnchorAccount } from '@helium/helium-react-hooks'
 import { HeliumEntityManager } from '@helium/idls/lib/types/helium_entity_manager'
 import { usePublicKey } from './usePublicKey'
 
 const type = 'keyToAssetV0'
+export type KeyToAssetV0 = IdlAccounts<HeliumEntityManager>['keyToAssetV0']
```

Comment:
Weird that this is exported from here but not used here?

Reply:
I mirrored what was going on with `useMobileInfo` and `useIotInfo`